lab2.js

```diff
@@ -127,12 +173,23 @@ function variablify(string) {
   // you might want to use these string methods:
   //  split(), charAt(), toUpperCase()
   // and this array method: join()
+  var stringToArray = string.split(" ");
+  var newArray = [];
+  newArray.push(stringToArray[0]);
+  for (var i = 1; i <= stringToArray.length; i++) {
```

Comment:
This should be < not <= , because the last element in the array is at array.length - 1.